hw_diag/diagnostics/mac_diagnostics.py

```diff
@@ -1,27 +1,14 @@
+from hm_pyhelper.diagnostics import DiagnosticsReport
 from hm_pyhelper.miner_param import get_mac_address
 from hm_pyhelper.diagnostics.diagnostic import Diagnostic
 
 
-INTERFACE_MAPPINGS = [
-  {
-    'key': 'E0',
-    'friendly_key': 'eth_mac_address',
-    'mac_filepath': '/sys/class/net/eth0/address'
-  },
-  {
-    'key': 'W0',
-    'friendly_key': 'wifi_mac_address',
-    'mac_filepath': '/sys/class/net/wlan0/address'
-  }
-]
-
-
 class MacDiagnostic(Diagnostic):
     def __init__(self, key, friendly_key, mac_filepath):
```

Comment:
Missing type hints

Reply:
Now, all the methods of the all diagnostics are type hinted.